documentation/app/template-remote-cluster.md

```diff
@@ -0,0 +1,187 @@
+## Bring your own cluster (remote deployments)
+
+This guide explains how to deploy application workloads to a remote OpenShift cluster while keeping Tekton Pipelines and webhooks on the host (RHDH) cluster. The GitOps application is split into two Argo CD apps:
+
+- app: Deploys the application manifests to the destination cluster (host or remote)
+- app-tekton: Manages Tekton resources on the host cluster so all PipelineRuns execute there
+
+### High-level behavior
+- Tekton webhooks and PipelineRuns stay on the host cluster.
```

Comment:
It would be usefull to define here too -> host vs remote cluster and what that means